SPECS/plexus-utils/plexus-utils.spec

```diff
@@ -20,9 +20,9 @@ Distribution:   Mariner
 
 Name:           plexus-utils
 Version:        3.3.0
```

Comment:
Latest is 3.5.1
https://github.com/codehaus-plexus/plexus-utils/releases/tag/plexus-utils-3.5.1

Reply:
For all plexus stuff we need a specific version as we are using older version of the `javapackages-bootstrap` and `xmvn` too.